var/spack/repos/builtin/packages/giraph/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Giraph(Package):
+    """Apache Giraph is an iterative graph processing system built
+    for high scalability."""
+
+    homepage = "https://giraph.apache.org/"
+    url      = "https://downloads.apache.org/giraph/giraph-1.0.0/giraph-dist-1.0.0-src.tar.gz"
+    list_url = "https://downloads.apache.org/giraph/"
+    list_depth = 1
+
+    version('1.2.0', sha256='6206f4ad220ea42aa0c4abecce343e36026cf9c6e0a2853f1eb08543da452ad1')
+    version('1.1.0', sha256='181d94b8198c0f312d4611e24b0056b5181c8358a7ec89b0393661736cd19a4c') 
+
+    depends_on('maven', type='build')
+    depends_on('java@8', type=('build', 'run'))
+
+    def install(self, spec, prefix):
+        mvn = which('mvn')
+        mvn('clean', 'package', '-DskipTests')
+        ver = self.version.up_to(3)
```

Comment:
It doesn't seem like this line is needed

Reply:
Fixed in https://github.com/spack/spack/pull/17790/commits/3a0e5cb35a9ac444096790af72dc1ed7382e7408